src/pages/Login.vue

```diff
@@ -416,7 +414,7 @@ export default {
       this.resendOTPTimer = seconds;
       const interval = setInterval(() => {
         this.resendOTPTimer--;
-        if (!this.resendOTPTimer) clearInterval(interval);
+        if (!this.resendOTPTimer || !this.requestedOtp) clearInterval(interval);

```

Comment:
a better solution would be to call clearInterval as soon as the `requestedOtp` value changes to false (based on its watcher) because here there is still a possibility that one might see resend otp = -1 - very slim chance but possible

Reply:
Okay I have made changes to it now, it will clear the interval as soon as the phone number is invalid.

Reply:
that is going to work in every case.